### ISSUE TITLE
link to Drupal.org project

### ISSUE THREAD

**Issue by frob** [ejanus/og_domain_access#10]
Where can I get this project on drupal.org

**Comment by ejanus** [comment]
You cannot. I have not submitted it or even bothered to try and get vetted
yet. Maybe I will soon.

*Kindest Regards,*

*Erek Janus*

**Comment by frob** [comment]
As someone who has gone through the vetting process, do it now, do not wait. It will take time, and they will help identify areas that need cleanup.

**Comment by ejanus** [comment]
Thanks, Frank! I'll keep that in mind.

Have you had a chance to give the module a try? =)

*Kindest Regards,*

*Erek Janus*

**Comment by frob** [comment]
Nope, I don't have a need for it. I just clicked the link in irc to see what it was.
